src/FsPickler.Json/FsPickler.Json.fsproj

```diff
@@ -114,32 +112,36 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="mscorlib" />
-    <Reference Include="FSharp.Core, Version=$(TargetFSharpCoreVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <Private>False</Private>
-    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Numerics" />
     <ProjectReference Include="..\FsPickler\FsPickler.fsproj">
       <Name>FsPickler</Name>
       <Project>{3ad987bf-f0fa-40bd-9dae-4af3a5fe9cc7}</Project>
       <Private>True</Private>
     </ProjectReference>
   </ItemGroup>
+  <Import Project="..\..\.paket\paket.targets" />
   <Choose>
-    <When Condition="$(TargetFrameworkVersion) != 'v3.5'">
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v3.5'">
+      <ItemGroup>
+        <Reference Include="FSharp.Core">
+          <HintPath>..\..\packages\FSharp.Core\lib\net20\FSharp.Core.dll</HintPath>
+          <Private>True</Private>

```

Comment:
It's crucial that these be ``<Private>False</Private>``.  This would be done by [making sure that ``copy_local: false`` is set](https://fsprojects.github.io/Paket/references-files.html#copy_local-settings) for each of the paket.references under the ``src`` directory

This is because you are using 4.0.0.1 when building under ``src``, but 4.1.17 when building under ``tests``